x/mint/keeper/keeper.go

```diff
@@ -95,6 +95,24 @@ func (k Keeper) MintCoins(ctx context.Context, newCoins sdk.Coins) error {
 	return k.bankKeeper.MintCoins(ctx, types.ModuleName, newCoins)
 }
 
+// BurnDeadAccount take all balances from dead account then burn.
+func (k Keeper) BurnDeadAccount(ctx context.Context) error {
+	deadBalances := k.bankKeeper.GetAllBalances(ctx, types.EvmDeadAddr)
+	if deadBalances.Empty() {
+		return nil
+	}
+	err := k.bankKeeper.SendCoinsFromAccountToModule(ctx, types.EvmDeadAddr, "gov", deadBalances)
+	if err != nil {
+		return err
+	}
+
```

Comment:
@hieuvubk Do we need to transfer the coins the the gov module before burning

Reply:
yes cause we need to burn from an module account have `Burner` permission

Reply:
It works but will cause a state breaking since `mint` module account already set in store. That why I used other module have `Burner` perm.
But we can go with an upgrade handler wdut @facs95